regression/cbmc/same-object-01/test-no-cp.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--no-simplify --no-propagation
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+^warning: ignoring
+--
+Check that __CPROVER_same_object() is true when given a pointer variable, and an
```

Comment:
Would it be worth it check that an offset into the object also maintains the same object property?

Reply:
Added